tests/wikidata/test_check_query.py

```diff
@@ -206,12 +201,9 @@ def test_all_queries(tree, expected):
 
 
 # MARK: execute
-
-
 def test_execute(a_query):
     with pytest.raises(QueryExecutionException) as err:
         _ = execute(a_query, 1, None, 0)
-
```

Comment:
Checking this now, @meerhaasham1! Some of these I'll add back in as I think that having a space after a `with` indentation makes it more readable :) I'll make the change 😊